src/Data/Neuroglia.Data/DomainException.cs

```diff
@@ -353,6 +352,28 @@ public static DomainArgumentException ArgumentMustBeEarlierThan(string argumentN
             return new DomainArgumentException(StringExtensions.Format(LocalizedResources.exception_domain_argumentmustbeearlierthan, argumentName, time), argumentName);
         }
 
+        /// <summary>
+        /// Creates a new <see cref="DomainArgumentException"/> for an argument that must be lower than the specified time
```

Comment:
> ... must be lower than the specified time ...

I would replace with:

`must be lower than the specified date and time`

Reply:
Should I write : 

- `time` when the argument name is time and `date` when the argument name is date
- `date or time` to match booth case without changing the comment between overload
- `date and time` like you say